packages/synapse-react-client/src/mocks/msw/handlers/asyncJobHandlers.ts

```diff
@@ -75,10 +81,13 @@ export function generateAsyncJobHandlers<
         const responseObject: TResponseBody =
           typeof response === 'function' ? response(request) : response
 
+        const jobState: AsynchJobState =
+          serviceSpecificEndpointResponseStatus < 400 ? 'COMPLETE' : 'FAILED'
+
         return res(
-          ctx.status(201),
+          ctx.status(200),
```

Comment:
This looks like a bug but it's not, should probably add a comment
```suggestion
          // This endpoint returns a successful status code regardless of the job status
          ctx.status(200),
```